vllm/v1/engine/core.py

```diff
@@ -31,6 +32,18 @@
 from vllm.v1.serial_utils import MsgpackDecoder, MsgpackEncoder
 from vllm.v1.structured_output import StructuredOutputManager
 from vllm.version import __version__ as VLLM_VERSION
+if not current_platform.is_tpu():
+    from vllm.v1.structured_output import StructuredOutputManager
+else:
+
+    class StructuredOutputManager:
+
+        def __init__(self, *args, **kwargs):
+            pass
+
+        def __getattr__(self, name):
+            return lambda *args, **kwargs: None
```

Comment:
I really hate this cheap workaround. Is anyone working on properly addressing this @njhill @russellb ?

Reply:
I'll push something now

Reply:
Here's an alternative: https://github.com/vllm-project/vllm/pull/14575